beard/clustering/blocking_funcs.py

```diff
@@ -368,14 +368,13 @@ def block_last_name_first_initial(X):
         array is the same as in the ``X`` input parameter.
     """
     def last_name_first_initial(name):
```

Comment:
The docstring of this function says "The names are normalized before assigning to a block.": we should either respect that or remove that comment from the docstring.